scanner/src/main/kotlin/scanners/fossid/FossId.kt

```diff
@@ -186,6 +196,23 @@ class FossId(
 
         log.info { "waitForResult parameter is set to '$waitForResult'" }
 
+        val namingProjectPattern = fossIdScannerOptions["namingProjectPattern"]
+        val namingScanPattern = fossIdScannerOptions["namingScanPattern"]
+
+        val namingConventionVariables = fossIdScannerOptions
+            .filterKeys { it.startsWith(NAMING_CONVENTION_VARIABLE_PREFIX) }
+            .mapKeys { it.key.substringAfter(NAMING_CONVENTION_VARIABLE_PREFIX) }
+
+        namingProjectPattern?.let {
+            log.info { "Naming pattern for projects is $namingProjectPattern." }
+        }
+
+        namingScanPattern?.let {
+            log.info { "Naming pattern for scans is $namingScanPattern." }
+        }
```

Comment:
Nit: Maybe move these straight to the variable assignment above for more compact code?